src/app/attendees/attendees.component.ts

```diff
@@ -35,82 +50,64 @@ export class AttendeesComponent {
     this.hideConfig = !(this.hideConfig);
   }
 
-  tokenProvided(token: string): void {
-    this.token = token;
-    this.fetchMettupEvents();
-  }
-
   eventIdProvided(newEventId: number): void {
     this.eventId = newEventId;
     this.toggleConfig();
     this.fetchAttendees();
   }
 
   fetchMettupEvents(): void {
-    if (this.token.length != 0) {
-      this.attendeeService.getMettupEvents(this.token)
-        .then(meetupEvents => this.meetupEvents = meetupEvents);
-    }
+    this.attendeeService.getMeetupEvents()
+      .then(meetupEvents => this.meetupEvents = meetupEvents);
   }
 
   fetchAttendees(): void {
-    if (this.eventId > 0 && this.token.length != 0) {
-      this.attendeeService.getAttendees(this.eventId, this.token)
+    if (this.eventId > 0) {
+      this.attendeeService.getAttendees(this.eventId)
         .then(attendees => this.attendees = attendees);
     }
   }
 
   randomAttendee(): void {
-    if (this.attendees.length == 0) {
+    if (this.attendees.length === 0) {
       alert('Patience, my young apprentice');
-    } else if (this.attendees.length == this.usedIndexes.length) {
+    } else if (this.attendees.length === this.usedIndexes.length) {
       alert('Everyone was already choosen');
     } else {
       let winner = this.attendees[this.randomAttendeeIndexWithoutRepetition()];
-      this.attendeeService.getMemberFull(winner, this.token)
+      this.attendeeService.getMemberFull(winner)
         .then(memberFull => {
-          if (memberFull.photo) {
```

Comment:
Why this part is removed? If I remember correctly Meetup API is not perfect and sometimes doesn't return `highres_link` to a photo so there is a need to use a fallback.